ledger/daml-on-sql/src/main/scala/on/sql/Main.scala

```diff
@@ -7,7 +7,7 @@ import java.util.concurrent.Executors
 import com.daml.ledger.resources.ResourceContext
 import com.daml.cliopts.GlobalLogLevel
 import com.daml.platform.sandbox.config.{PostgresStartupMode, SandboxConfig}
-import com.daml.platform.sandbox.SandboxServer
+import com.daml.ledger.sandbox.SandboxServer
```

Comment:
I suggest putting these imports in their alphabetic place

Reply:
Alright, I usually let IJ do its magic. Looking forward to us implementing a automagic tool part of the fmt process

Reply:
done @mziolekda